mcs/class/corlib/System.Globalization/CultureInfo.cs

```diff
@@ -1123,6 +1124,39 @@ static Exception CreateNotFoundException (string name)
 			}
 		}
 
+		static CultureInfo s_UserPreferredCultureInfoInAppX;
+
+		delegate void OnCultureInfoChangedDelegate ([MarshalAs(UnmanagedType.LPWStr)] string language);
+
+		[DllImport("__Internal")]
```

Comment:
Nit, we could have done this with an icall via `extern` without the `DllImport` but I think this is fine.

Reply:
I thought this was easier because this way I don't have to worry about GC memory/marshaling on the native side. P/invoke does it for me.